tests/testthat/test-cache.R

```diff
@@ -21,13 +21,12 @@ describe("cache function", {
      lapply(list(1:10, 1:20), function(ids) {
        # Populate the cache and make sure that the results are equal
        expect_equal(dim(bd <- batch_huge_data(ids)), dim(cached_fcn(ids)))
-       # Make sure we're reading from cache and it's fast now!
-       takes_less_than(1)(tmp <- cached_fcn(ids))
+       tmp <- cached_fcn(ids)
        # And the results are still correct
        expect_equal(dim(bd), dim(tmp))
      })
      # And now everything is so cached
-     takes_less_than(1)(tmp <- cached_fcn(1:20))
```

Comment:
Good idea, though this does remove the test that things are actually cached and I am bit nervous about the timing here. ...But another time.